pages/web_cloud/domains/dns_zone_dkim/guide.pl-pl.md

```diff
@@ -207,22 +209,216 @@ Upewnij się również, czy domena, której chcesz używać do obsługi e-maili
 
 ![email](images/dns-dkim-domain.png){.thumbnail}
 
+#### Kompletna konfiguracja DKIM <a name="firststep"></a>
+
 Aby skonfigurować DKIM, wejdź na stronę <https://api.ovh.com/console/> WWW, zaloguj się przyciskiem `Login`{.action} w prawym górnym rogu i wprowadź dane identyfikacyjne OVHcloud.
 
 > Skorzystaj z naszego przewodnika ["Dowiedz się, jak korzystać z API OVHcloud"](/pages/manage_and_operate/api/first-steps), jeśli nigdy wcześniej nie korzystałeś z API.
 
-Przejdź do sekcji `/e-mail/exchange` (oferty Exchange) lub `/e-mail/pro` (oferta E-mail Pro) API i wpisz "dkim" w polu `Filter`, aby wyświetlić tylko API powiązane z DKIM.
+Przejdź do sekcji`/email/domain/`(oferty E-mail (MXplan))`/email/exchange` (oferty Exchange) lub`/email/pro` (oferta E-mail Pro) interfejsów API i wpisz "dkim" w polu `Filter`, aby wyświetlić tylko API związane z DKIM.
 
-![email](images/dns-dkim-api01.png){.thumbnail}
+Kliknij zakładkę odnoszącą się do Twojej oferty:
 
-#### Kompletna konfiguracja DKIM <a name="firststep"></a>
+> [!tabs]
+> **E-maile (MX Plan)**
+>>
+>> ![email](obrazy/dns-dkim-api01.png){.thumbnail}
```

Comment:
`obrazy` instead of `images`
This needs to be search & replaced on the whole file